src/app/child-dev-project/attendance/add-day-attendance/roll-call/roll-call.component.ts

```diff
@@ -116,4 +120,43 @@ export class RollCallComponent implements OnInit {
   isFinished(): boolean {
     return this.currentIndex >= this.entries?.length;
   }
+
+  save() {
+    const dialogRef = this.confirmationDialog.openDialog(
+      $localize`:Save & Next confirmation title: Save & Exit`,
+      $localize`:Save & Exit confirmation text:Are you sure you want to save and exit?`
+    );
+
+    dialogRef.afterClosed().subscribe((confirmed) => {
+      if (confirmed) {
+        this.goToNextParticipant(this.entries?.length);
+      }
+    });
+  }
+
+  skip() {
```

Comment:
I don't think it's necessary to have a popup on this button. This could become very annoying when a user has to skip multiple children in a single roll call.

Reply:
Yes! you are right, I will remove it. Actually, it was written in the description so I added the popup.

Reply:
I think the description of the issue meant that in the popup of the `save & exit` button you should display, how many participants will be skipped if the roll-call is exited now.

E.g. "Are you sure you want to exit, save and skip the <x> participants"

Reply:
Okay, I got it now. Thanks!